journalbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,186 @@
+----
++
+If no paths are specified, {beatname_uc} reads from the default journal.
+
+. Set the <<{beatname_lc}-seek,`seek`>> option to control the position where
+{beatname_uc} starts reading the journal. The available options are `head`,
+`tail`, and `cursor`. The default is `cursor`, which means that on first read,
+{beatname_uc} starts reading at the beginning of the file, but continues reading
+at the last known position after a reload or restart. For more detail about
+the settings, see the reference docs for the
+<<{beatname_lc}-seek,`seek` option>>. 
+
+. (Optional) Set the <<{beatname_lc}-include-matches,`include_matches`>> option
+to filter entries in journald before collecting any log events. This reduces the
+number of events that {beatname_uc} needs to process. For example, to fetch only
+Redis events from a Docker container tagged as `redis`, use: 
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- paths: []
+  include_matches:
+    - "CONTAINER_TAG=redis"
+    - "_COMM=redis"
+----
+
+include::../../libbeat/docs/step-configure-output.asciidoc[]
+
+include::../../libbeat/docs/step-configure-kibana-endpoint.asciidoc[]
+
+include::../../libbeat/docs/step-configure-credentials.asciidoc[]
+
+include::../../libbeat/docs/step-test-config.asciidoc[]
+
+include::../../libbeat/docs/step-look-at-config.asciidoc[]
+
+
+[id="{beatname_lc}-template"]
+=== Step 3: Load the index template in Elasticsearch
+
+include::../../libbeat/docs/shared-template-load.asciidoc[]
+
+[id="{beatname_lc}-starting"]
+=== Step 4: Start {beatname_uc}
+
+Start {beatname_uc} by issuing the appropriate command for your platform. If you
+are accessing a secured Elasticsearch cluster, make sure you've configured
+credentials as described in <<{beatname_lc}-configuration>>.
+
+NOTE: If you use an init.d script to start {beatname_uc} on deb or rpm, you can't
+specify command line flags (see <<command-line-options>>). To specify flags,
+start {beatname_uc} in the foreground.
+
+*deb and rpm:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo service {beatname_lc} start
+----------------------------------------------------------------------
+
+*linux:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo chown root {beatname_lc}.yml <1>
+sudo ./{beatname_lc} -e
+----------------------------------------------------------------------
+<1> You'll be running {beatname_uc} as root, so you need to change ownership
+of the configuration file, or run {beatname_uc} with `--strict.perms=false`
+specified. See
+{libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
+in the _Beats Platform Reference_.
+
+{beatname_uc} is now ready to send journal events to the defined output.
+
+[[view-kibana-dashboards]]
+=== Step 5: View your data in Kibana
+
+There are currently no example dashboards available for {beatname_uc}.
```

Comment:
I have added one in the PR I have linked.

Reply:
@kvch I'm assuming that there are no dashboards for 6.5 though, correct? I want to get the 6.5 version of the docs finalized.